tensorflow_decision_forests/contrib/temporal_feature_processor/temporal_feature_processor/implementation/pandas/operators/tests/assign/assign_test.py

```diff
@@ -0,0 +1,37 @@
+from absl.testing import absltest
+
+import test_data
+from temporal_feature_processor.implementation.pandas.operators import PandasAssignOperator
+
+
+class AssignOperatorTest(absltest.TestCase):
+  base_data_dir = "temporal_feature_processor/tests/operators/assign/data"
+
+  def setUp(self) -> None:
+    self.operator = PandasAssignOperator()
+
+  def test_different_index(self) -> None:
+    self.assertRaises(IndexError, self.operator,
```

Comment:
Can you use "assertRaisesRegex" to ensure the error message is correct.

Reply:
Definitely!